frontend/src/scenes/funnels/FunnelCanvasLabel.tsx

```diff
@@ -21,20 +20,27 @@ export function FunnelCanvasLabel(): JSX.Element | null {
 
     return (
         <div className="funnel-canvas-label">
-            {allFilters.display === FUNNEL_VIZ && (
+            {allFilters.display === ChartDisplayType.FunnelViz && (
                 <>
                     <span className="text-muted-alt">Total conversion rate: </span>
-                    <span>{totalConversionRate}%</span>
+                    <span>{conversionMetrics.totalRate}%</span>
                     <span style={{ margin: '2px 8px', borderLeft: '1px solid var(--border)' }} />
                 </>
             )}
-            <span className="text-muted-alt">Average time to convert: </span>
+            {allFilters.display === ChartDisplayType.FunnelsTimeToConvert && (
+                <>
+                    <span className="text-muted-alt">Total time to convert: </span>
+                    <span>{humanFriendlyDuration(conversionMetrics.sum)}</span>
+                    <span style={{ margin: '2px 8px', borderLeft: '1px solid var(--border)' }} />
+                </>
+            )}
+            <span className="text-muted-alt">Mean time to convert: </span>
```

Comment:
I think we should use "Average", it's more common and consistent with the rest of the views

Reply:
I think there was some confusion whether "average" meant mean or median, and I suggested just renaming it to mean so we could avoid adding a tooltip to do that which seemed like overkill but curious to know if there are other simple ways for it? Like `Average (mean) time to convert`?

Reply:
Thanks for that context Li! Curious how this came along, I wouldn't have seen median being understood under average, but it might be for some users? I would actually advocate for the info icon with the tooltip where we also clarify how this is computed

Reply:
Agreed! I'll keep the wording as average and bring that explanation into a popover. It might also be helpful to include in that popover other statistics common to histograms such as Q1, median, Q3, total which provides more insight into how the data is broken up.

Reply:
Gonna tackle non-avg stats in another PR